plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2738,6 +2738,7 @@ workspace.textures.new=Create new texture
 workspace.textures.import=Import textures
 workspace.textures.edit_selected=Edit selected ...
 workspace.textures.duplicate_selected=Duplicate selected ...
+workspace.textures.replace_selected=Replace selected ...
```

Comment:
I have noticed we use space before ... to action requires user input, but there should be no space. I have corrected others on master so merge in this PR and also remove space for this entry